src/redux/books/Books.js

```diff
@@ -0,0 +1,36 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const initialState = [];
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.info];
+    case REMOVE_BOOK:
+      return [...state.filter((book) => book.title !== action.info.title)];
```

Comment:
- [ ] (OPTIONAL) It would be better to filter a book by `id` instead of `title` since the `ids` of each book is unique but 2 different books can have the same `title` 👍